src/openpgp.js

```diff
@@ -158,7 +158,7 @@ export function decryptKey({ privateKey, passphrase }) {
 export function encrypt({ data, publicKeys, privateKeys, passwords, filename, armor=true }) {
   checkData(data); publicKeys = toArray(publicKeys); privateKeys = toArray(privateKeys); passwords = toArray(passwords);
 
-  if (!nativeAEAD() && asyncProxy) { // use web worker if web crypto apis are not supported
+  if (asyncProxy) { // use web worker when available
```

Comment:
This will probably cause a performance regression as well for large files. Especially if transferable objects are not activated.